canonical-web-and-design/maas.io
# Handle redirects in discourse

## Done

Handle redirects in discourse and not in the app itself for specific urls

## QA

- `./run`
- try to access a few urls removed from the file.
- Make sure they redirect well.
- eg. https://maas-io-canonical-web-and-design-pr-454.run.demo.haus/docs/installconfig-network-ssl -> https://maas-io-canonical-web-and-design-pr-454.run.demo.haus/docs/ssl

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2371
